gopdf.go

```diff
@@ -1205,73 +1252,45 @@ func infodate(t time.Time) string {
 	return ft
 }
 
-// SetAlpha sets transparency.
+// SetTransparency sets transparency.
 // alpha: 		value from 0 (transparent) to 1 (opaque)
 // blendMode:   blend mode, one of the following:
 //          		Normal, Multiply, Screen, Overlay, Darken, Lighten, ColorDodge, ColorBurn,
 //          		HardLight, SoftLight, Difference, Exclusion, Hue, Saturation, Color, Luminosity
-func (gp *GoPdf) SetAlpha(alpha float64, blendModeStr string) error {
-	if alpha < 0.0 || alpha > 1.0 {
-		return errors.Unwrap(fmt.Errorf("alpha value (0.0 - 1.0) is out of range: %.3f", alpha))
-	}
-
-	blendMode, err := getBlendMode(blendModeStr)
+func (gp *GoPdf) SetTransparency(transparency Transparency) error {
```

Comment:
Function arguments are not the same as in README.md.
```GO
//README.md
SetTransparency (alpha float64, blendMode string)
```
```GO
//code
SetTransparency (transparency Transparency) 
```


By the way, I think the arguments in README.md seem easier to use.

Reply:
oh no, sorry
i'm so inattentif

Reply:
1 min and i will fix it

Reply:
now fixed
please look 🙏